MIP113/MIP113.md

```diff
@@ -389,7 +389,9 @@ The AVC decisions that moderators have elected authority to make without the off
 
 #### 5.2.1
 
-To become recognized as an Unaffiliated AVC Member, an Ecosystem Actor must post a recognition submission message publicly on the Maker Governance Forum.
+To become recognized as an Unaffiliated AVC Member, an Ecosystem Actor must post a recognition submission message publicly on the Maker Governance Forum, as described *5.2.1.1*, *5.2.1.2*, *5.2.1.3*, and *5.2.1.4*.
+
+A recognized AVC member can lose their recognized role by either failing to fulfill the requirements, as described in *5.2.4.4*, or by submitting a written relinquishment as described in 5.2.1.5.
```

Comment:
I realized there's a number of other references that are missing proper formatting elsewhere in the document. Instead of fixing them all now, and in the interest of keeping the PR easy to read, I think we can do another pass at a later time. What do you think?

Reply:
That's also fine. Please consider adding "in" on line 392 to make it more readable.

Reply:
Ah, sorry. Missed that one. Yeah, I'm committing your suggestion. Thank you!